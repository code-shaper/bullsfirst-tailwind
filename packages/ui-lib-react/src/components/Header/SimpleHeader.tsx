import * as React from 'react';

interface SimpleHeaderProps {
  children?: React.ReactNode;
}

export function SimpleHeader({ children }: SimpleHeaderProps) {
  return (
    <header className="relative">
      <nav aria-label="Top">
        <div className="bg-primary-default">
          <div className="max-w-7xl mx-auto h-12 px-4 flex items-center justify-between">
            <span className="flex-1 text-base font-medium text-white">
              {children}
            </span>
          </div>
        </div>
      </nav>
    </header>
  );
}
