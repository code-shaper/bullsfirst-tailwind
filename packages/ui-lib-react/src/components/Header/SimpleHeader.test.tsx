import { render } from '../../test/test-utils';
import { SimpleHeader } from './SimpleHeader';

describe('<SimpleHeader />', () => {
  test('renders correctly', () => {
    const { asFragment } = render(<SimpleHeader>Bullsfirst</SimpleHeader>);
    expect(asFragment()).toMatchSnapshot();
  });
});
