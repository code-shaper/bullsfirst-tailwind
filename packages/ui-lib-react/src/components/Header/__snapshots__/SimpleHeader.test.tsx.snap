// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SimpleHeader /> renders correctly 1`] = `
<DocumentFragment>
  <header
    class="relative"
  >
    <nav
      aria-label="Top"
    >
      <div
        class="bg-primary-default"
      >
        <div
          class="max-w-7xl mx-auto h-12 px-4 flex items-center justify-between"
        >
          <span
            class="flex-1 text-base font-medium text-white"
          >
            Bullsfirst
          </span>
        </div>
      </div>
    </nav>
  </header>
</DocumentFragment>
`;
